.github/workflows/e2e-test-manual.yml

```diff
@@ -37,6 +37,7 @@ on:
       osImage:
         description: "Full name of OS image (CSP independent image version UID)."
         type: string
+        default: "main latest"
```

Comment:
nit: For me it sounds clearer to call this `debug-latest`, but feel free to ignore.